pkg/manager/paths/paths_gallery.go

```diff
@@ -15,6 +20,31 @@ func (gp *galleryPaths) GetExtractedPath(checksum string) string {
 	return filepath.Join(config.GetCachePath(), checksum)
 }
 
+func GetGthumbCache() string {
+	return filepath.Join(config.GetCachePath(), thumbDir)
+}
+
+//return a string that can be added to filepath.Join to implement directory depth
+func getIntraDir(checksum string) string {
+	if thumbDirDepth == 0 {
+		return "" // filepath.Join ignores empty elements
+	}
+	intraDir := checksum[0:thumbDirLength]
+	for i := 1; i < thumbDirDepth; i++ {
+		intraDir = filepath.Join(intraDir, checksum[thumbDirLength*i:thumbDirLength*(i+1)])
+	}
+	return intraDir
+}
```

Comment:
Can this be moved into a general utility function in the utils module with a little bit more detail in the comment?
Line 34 could use a comment explaining what it is doing.

Reply:
done